### PR TITLE
feat: Explore Videos Page

### LINKED ISSUE #5: Todo: Explore Videos Page

### PR DESCRIPTION
**Pull request by coldpigli**
What's New:

1) Added Filter UI
2) Added Video Listing with VideoCard
3) Made a custom dropdown
4) Made a context for providing Videos

Preview

Desktop:
image

Mobile View:
image

### CODE REVIEW

File: src/components/CategoryList/CategoryList.jsx [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-  if(isLoading===false && errorFlag === false){
-     categories = responseData?.categories
-  }
+  const {categories, isLoading} = useCategories();
 
   return (
     <div className={`${styles.categoryList} flex wrap`}>
       {
-        categories.map((item)=><CategoryCard category={item}/>)
+        categories.slice(1).map((item)=><CategoryCard category={item}/>)
```
Comment (nishasen) [inline src/components/CategoryList/CategoryList.jsx:13]: I like your this part of the code

File: src/contexts/VideoContext/videoReducer.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const videoReducer = (state, action) => {
+    switch (action.type) {
+        case "GET_VIDEOS":
+            console.log("payload",action.payload)
```
Comment (rahulcs754) [inline src/contexts/VideoContext/videoReducer.js]: after done work remove console log

**Review by rahulcs754 — APPROVED** [review]
The code looks goods and clean 
give just small suggestion

File: src/contexts/VideoContext/videoReducer.js (hunk 2 above)
Comment (nishasen) [inline src/contexts/VideoContext/videoReducer.js]: remove the console.logs

**Review by nishasen — APPROVED** [review]
Your code looks good and clean, liked how you are making use of hooks for api call

**Pull request merged**